assets/js/components/ChargingSessionModal.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+	<Teleport to="body">
+		<div
+			id="sessionDetailsModal"
+			class="modal fade text-dark"
+			tabindex="-1"
+			role="dialog"
+			aria-hidden="true"
+		>
+			<div class="modal-dialog modal-dialog-centered modal-dialog-scrollable" role="document">
+				<div v-if="session != undefined" class="modal-content">
+					<div class="modal-header">
+						<h5>{{ $t("session.title") }}</h5>
+						<button
+							type="button"
+							class="btn-close"
+							data-bs-dismiss="modal"
+							aria-label="Close"
+						></button>
+					</div>
+					<div class="modal-body">
+						<table class="table">
+							<tbody>
+								<tr>
+									<th colspan="2"></th>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.csv.created") }}
+									</th>
+									<td>
+										{{ fmtFullDateTime(new Date(session.created), false) }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.csv.finished") }}
+									</th>
+									<td>
+										{{ fmtFullDateTime(new Date(session.finished), false) }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.energy") }}
+									</th>
+									<td>
+										{{ fmtKWh(session.chargedEnergy * 1e3) }}
+									</td>
+								</tr>
+								<tr>
+									<th></th>
+									<td></td>
+								</tr>
```

Comment:
I'd remove these empty lines. If we want to have logically grouped blocks we should label them. But for the start I think a simple list is fine.

Reply:
A general topic regarding the order. I'd keep the order of the values in the modal closer to the order we picked in the table:
- Loadpoint
- Vehicle (+ Odometer)
- Energy (+ Meter)
- Period (Start/Finish)